webui/src/js/utils/k8s-domain-resource-generator.js

```diff
@@ -247,7 +248,7 @@ define(['models/wkt-project', 'utils/k8s-domain-configmap-generator', 'js-yaml',
     }
 
     function getServerPodForCluster(cluster) {
-      const serverPod = _getServerPod(getJavaOptionsForCluster(cluster), getUserMemArgsForCluster(cluster), getKubernetesResourcesForCluster(cluster)) || {};
+      const serverPod = _getServerPod(getJavaOptionsForCluster(cluster), getUserMemArgsForCluster(cluster), getKubernetesResourcesForCluster(cluster), null) || {};
```

Comment:
If you remove the ugly `null` argument, then the receiving function will simply get `undefined` for the argument value.  If it makes you feel better, you can add a default value for that argument in the function argument list.  That is a more JavaScript-way of doing this.

```
const serverPod = _getServerPod(getJavaOptionsForCluster(cluster), getUserMemArgsForCluster(cluster), getKubernetesResourcesForCluster(cluster)) || {};

...
function _getServerPod(javaOptions, userMemArgs, resources, nodeSelector = null) {
...

```